EddiInaraService/IInaraService.cs

```diff
@@ -1,26 +1,20 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 
 
 namespace EddiInaraService
 {
     public interface IInaraService
     {
-        // Variables
-        string commanderName { get; }
-        string commanderFrontierID { get; }
-        DateTime lastSync { get; }
-
         // Background Sync
-        void Start(bool gameIsBeta = false, bool eddiIsBeta = false);
+        void Start(bool eddiIsBeta = false);
```

Comment:
Why conflate two variables that have different intentions?

Reply:
We're not. `gameIsBeta` simply isn't necessary because we're handling that case in the Inara Responder. https://github.com/EDCD/EDDI/blob/c98f2127478d586a91dbb8096c0e5bbe47b57eaa/InaraResponder/InaraResponder.cs#L105-L109.